### ISSUE TITLE
Toggling pause/play prone to error compared to native

### ISSUE THREAD

**Issue by evoactivity** [muxinc/media-chrome#688]
I've written a custom element for mpeg2-ts streams using mpegts.js 
- https://github.com/ektotv/mpegts-video-element
- https://www.npmjs.com/package/mpegts-video-element

When using native browser ui, toggling pause/play works as you'd expect. When using media chrome ui, toggling pause/play a few times causes the stream to end up buffering forever.

Repro is my repo, pull down, `pnpm install`, `pnpm dev`, use a CORS addon ([firefox](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/), [chrome](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf)) to skip cors issue with stream.

If you toggle the play/pause state on both video players you should see when used with media chrome you are more likely to cause a problem compared to the native ui.

I'm not really sure how to debug this.

**Comment by evoactivity** [comment]
That's what I thought it was doing. The demo for mpegts.js pauses and plays the stream as I expect. When I use `.pause()` and `.play()` from the console on the native video element, or when I use the native ui, or if I put it into pip mode it works as expected. 

I also wrote a method that would toggle the pause state every second and that works fine.

It seems when clicking the play button, if the stream is live it will try to set the current time to the very end of the current buffer, which I guess mpegts.js doesn't handle well.

---

I went looking at source code whilst writing this reply, [it doesn't just seem to seek to live, it does! ](https://github.com/muxinc/media-chrome/blob/2ebea974db52f5f00cc616012a7ede876ccc255c/src/js/controller.js#L482C11-L482C25)

I found `noautoseektolive` is an undocumented attribute for media-controller which fixes my issue.
